source/faq.txt

```diff
@@ -68,10 +68,12 @@ What Is the Difference Between "connectTimeoutMS", "socketTimeoutMS" and "maxTim
        never time out the socket. This option applies only to sockets that
        have already been connected.
    * - maxTimeMS
-     - `maxTimeMS <{+api+}/classes/FindCursor.html#maxTimeMS>`__ is the maximum
-       amount of time the server should wait for an operation to complete
-       after it has reached the server. If an operation runs over the
-       specified time limit, it returns a timeout error.
+     - `maxTimeMS <{+api+}/classes/FindCursor.html#maxTimeMS>`__ is only
+       passed to an individual operation in the driver or to a
```

Comment:
Suggestion:

For consistency with the other terms and to have the most important information first, maybe consider putting the definition ("...specifies the maximum amount of time...") before the fact that it can only be passed in an individual operation or to a cursor? This would also give you an opportunity to remove the passive voice.

For example:
"maxTimeMS is the maximum amount of time the server should wait for an operation... You can only pass it to an individual operation in the driver or to a cursor."